### PR TITLE
Add blog post for installing grafana on openshift 

### PR DESCRIPTION
**Pull request by david-martin**
Uses latest v5 version of operator.
I based the post on https://cloud.redhat.com/experts/o11y/ocp-grafana/ (which uses v4),
but with a kuadrant spin on it as it's for the kuadrant blog.

### CODE REVIEW

File: src/blog/grafana-on-openshift-for-kuadrant.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+kind: Grafana
+metadata:
+  labels:
+    dashboards: grafana
+  name: grafana
+  namespace: openshift-operators
+spec:
+  config:
+    auth:
+      disable_login_form: 'false'
```
[the remark this answers is not in the document]
Comment (david-martin) [reply src/blog/grafana-on-openshift-for-kuadrant.md:59]: i could login as root/secret with this option

File: src/blog/grafana-on-openshift-for-kuadrant.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    operators.coreos.com/grafana-operator.openshift-operators: ""
+  name: grafana-operator
+  namespace: openshift-operators
+spec:
+  channel: v5
+  installPlanApproval: Automatic
+  name: grafana-operator
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: grafana-operator.v5.15.1
```
[the remark this answers is not in the document]
Comment (david-martin) [reply src/blog/grafana-on-openshift-for-kuadrant.md:37]: I didn't realise it was optional
I'll remove it as installing the latest version sounds reasonable

File: src/blog/grafana-on-openshift-for-kuadrant.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (eguzki) [reply src/blog/grafana-on-openshift-for-kuadrant.md:58]: :shrug:

**Review by eguzki — APPROVED** [review]

**Comment by jsmolar** [comment]
- The token for Thanos DataSource is valid for a set amount of time. I wonder if you need to inform about this.
- User workload monitoring needs to be enabled as well: https://docs.openshift.com/container-platform/4.8/monitoring/enabling-monitoring-for-user-defined-projects.html
you can verify this by: `oc get cm/cluster-monitoring-config -n openshift-monitoring -ojsonpath='{.data.config\.yaml}'`

**Comment by david-martin** [comment]
Thanks for reviewing @jsmolar 
I've added a note about the token potentially expiring (with a link to openshift docs),
and a note about having user workload monitoring enabled (again, with a link to openshift docs).

**Review by jasonmadigan — APPROVED** [review]
looks good

**Pull request merged**